$ go get failed because of "cli" package dependency

When I tried to install this product with command `$ go get github.com/motemen/blogsync`, or `$ cd ${GOPATH}/src/github.com/motemen/blogsync && go build -o blogsync` after cloning this git repository, the following error has occurred, regardless of where the installation was tried, such as on my local machine(WindowsPowerShell) or on remote host(CircleCI-GoContainer).

```
C:\Users\***\go\src\github.com\motemen\blogsync\main.go:17:15: cannot use []cli.Command literal (type []cli.Command) as type []*cli.Command in assignment
C:\Users\***\go\src\github.com\motemen\blogsync\main.go:157:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
C:\Users\***\go\src\github.com\motemen\blogsync\main.go:158:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in array or slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
C:\Users\***\go\src\github.com\motemen\blogsync\main.go:159:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in array or slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
```

```
$ #!/bin/bash -eo pipefail
cd ${GOPATH}/src/github.com/motemen/blogsync && go build -o blogsync
# github.com/motemen/blogsync
./main.go:17:15: cannot use []cli.Command literal (type []cli.Command) as type []*cli.Command in assignment
./main.go:157:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
    cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
./main.go:158:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in array or slice literal:
    cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
./main.go:159:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in array or slice literal:
    cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)

Exited with code exit status 2
```

I guess that the dependency between `main.go` and `urfave/cli`, which is mentioned as one of imported packages in `main.go`, leads this error.  For your information, `urfave/cli` branched off in autumn 2019, and now there are two versions of that component, then `import` method written below may result in installation of `urfave/cli/v1`.
```
import (
	"github.com/urfave/cli"
)
```
cf. https://github.com/urfave/cli
cf. https://github.com/swaggo/swag/issues/556

So I want to let you know the phenomenon, and I hope you would fix this issue.

Best regards.

--- a/main.go
+++ b/main.go
@@ -154,9 +154,9 @@ var commandPost = cli.Command{
 	Name:  "post",
 	Usage: "Post a new entry to remote",
 	Flags: []cli.Flag{
-		cli.BoolFlag{Name: "draft"},
-		cli.StringFlag{Name: "title"},
-		cli.StringFlag{Name: "custom-path"},
+		&cli.BoolFlag{Name: "draft"},
+		&cli.StringFlag{Name: "title"},
+		&cli.StringFlag{Name: "custom-path"},
 	},
 	Action: func(c *cli.Context) error {
 		blog := c.Args().First()

